html/images/misc/android-apk-40x135.svg

```diff
@@ -0,0 +1,16 @@
+        <text id="Android-APK" font-family="Roboto-Medium, Roboto" font-size="16" font-weight="400" fill="#FFFFFF">
+            <tspan x="38" y="25">Android APK</tspan>
```

Comment:
All the SVG files on OFF should have the text converted to paths, otherwise it might not display as you expect.

e.g. in my browser in Ubuntu, I don't have the Roboto font, so I get a default font:

![image](https://user-images.githubusercontent.com/8158668/83551692-f9f31400-a508-11ea-9b6d-4f3c0b99062f.png)

Reply:
should be fixed